src/typings/linux-ca.d.ts

```diff
@@ -0,0 +1,10 @@
+declare module 'linux-ca' {
+  // eslint-disable-next-line import/no-duplicates
```

Comment:
I don't think this is necessary (you're not importing `node-forge` in a different module in the same typings file).